i18n.install

```diff
@@ -50,13 +64,162 @@ function i18n_update_last_removed() {
 }
 
 /**
- * Move book settings from variables to config.
+ * Empty hook.
  */
 function i18n_update_1000() {
-  // Migrate variables to config.
-  // $config = config('i18n.settings');
-  // $config->set('book_allowed_types', update_variable_get('book_allowed_types', array('book')));
-  // $config->save();
-  // Delete variables.
-  // update_variable_del('book_allowed_types');.
+  // Got removed.
```

Comment:
Might be better to explain why it got removed - that it contained just dummy code.